packages/clarity-js/src/data/upload.ts

```diff
@@ -252,5 +252,8 @@ function response(payload: string): void {
             // Upgrade current session to send back playback information
             clarity.upgrade(Constant.Auto);
             break;
+        case Constant.Action:
+            config.action && config.action(payload.split(" ")[1]);
```

Comment:
Since the response comes from a server and there's a potential for a bug -- we should check if the response has a space before referencing [1] to avoid null exceptions.